site/src/pug/includes/body.pug

```diff
@@ -40,7 +40,8 @@ include ./fork.pug
                 - difficultyTitle = 'Difficulty: ' + swag.difficulty;
                 div.item
                     div.title.flex
-                        h1=swag.name
+                        h1
+                          a.no-underline(href=swag.reference) #{swag.name}
```

Comment:
Currently, the `h1` color is taking the color of `a` tags. I suggest removing `.no-underline` class
```suggestion
                          a(href=swag.reference) #{swag.name}
```